src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/createnewwiki/CreateNewWikiPageObjectStep1.java

```diff
@@ -106,10 +107,16 @@ public CreateNewWikiPageObjectStep2 submit() {
     return new CreateNewWikiPageObjectStep2(driver);
   }
 
-  public CreateNewWikiLogInSignUpPageObject submitToLogInSignUp() {
+//  public CreateNewWikiLogInSignUpPageObject clickNextToSignIn() {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3ACommentedOutCodeLine)